app/components/autocomplete/autocomplete.js

```diff
@@ -212,6 +219,14 @@ export default class Autocomplete extends PureComponent {
                     ref={this.containerRef}
                     style={containerStyles}
                 >
+                    <AppSlashSuggestion
```

Comment:
Can I propose that the App Autocomplete only mounts when needed and do not mount the other ones when `appsTakenOver` is true as they will render `null`?

I think that will make this way more efficient

Reply:
I was mainly following the previous pattern.
My guess was that for some reason, it was better to have everything mounted (maybe mount cost? state cache?) than keep mounting and unmounting the needed components.
Did I understood that wrong? Do you think it is better to mount and unmount the components?

Reply:
So mounting components is not really a biggy but they do need to allocate memory. Now the connected components will compute every time anything in the redux state changes, even if they memoized, is still doing more than what is needed.

A fact is that we improve the post list rendering performance by more than 40% and most of it was because we only mounted the components that we needed.

If your component does lots of things when mounting then you could reconsider

Reply:
I have gone through this, and it seems viable to not mount the rest of the components, but the AppSlashSuggestion must be always active, because it is the responsible of updating the value that updates the `appsTakeOver` variable.

Reply:
I think this is fine, one thing I'm unsure of, will this require a minimum server version? are we checking for that or is it not required?